### ISSUE TITLE
[REC] Destruction Warlock - Rain of Fire recomendation with 2 targets

### ISSUE THREAD

**Issue by Sdw8** [Hekili/hekili#1513]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

With Eradication, Inferno, and Rain of Chaos talents, if only 2 active targets, Rain of Fire is always recommended. Up to two targets must use Chaos Bolt.

### How to Reproduce

1. Enter game as Destruction Warlock.
2. Take the Eradication, Inferno, and Rain of Chaos talents.
3. DPS 2 tagets.
4. See recomendations.

### Snapshot (Link)

https://pastebin.com/Rh78KACT

**Comment by Hekili** [comment]
Thanks for the report.

Have you simmed yourself in a 2 target scenario and looked at what your sim does?

**Comment by Sdw8** [comment]
2 target also uses Rain of Fire, but looking at a lot of logs, I see that everyone uses Chaos Bolt in 2 targets, even without Chaos . But at least with active Chaos, it seems to me that Chaos Bolt should be used, or am I wrong?

**Comment by Hekili** [comment]
This is the sort of thing to prove with data.  Logs help, but comparative sims would be useful for a direct comparison.  Looking at the sim priority, it's pretty explicitly written to use the AOE action list with 3 enemies by default, but 2 enemies with Tier 28 4pc.

`actions+=/call_action_list,name=aoe,if=active_enemies>2-set_bonus.tier28_4pc`

Want to link your sim?  I can run a few comparison sims.

**Comment by Sdw8** [comment]
https://www.raidbots.com/simbot/report/t3hbpvMAD3zXqWRYDNbFLK
I don't know how to change the priority to use Chaos Bolt :)

**Comment by Hekili** [comment]
So, I tried calling the AOE action list only with 3+ targets (despite the sim profile being written to fork at 2 targets with Tier 28), and it was a significant DPS loss.

I also tried having the AOE action list run with 2+ targets with Tier 28, but only calling Rain of Fire with 3+ targets.  It was also a significant DPS loss.

I'm considering adding a spec option that will force the 3 target requirement for the AOE action list, with a warning that it's not proven in sims but appears more consistent with sims.
